src/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Lifesoft Todo app</title>
+</head>
+
+<body>
+    <div class="container">
+        <div class="main-list">
+            <div class="item-container">
+                <div>
+                    <div class="today-content white-height">
+                        <span class="heading">Today's To Do list</span>
+                        <i class="fas fa-sync"></i>
+                    </div>
+                    <form class="add-new white-height">
+                        <input class="no-outline" maxlength="128" id="new-item" type="text"
+                            placeholder="Add to your list ..." />
+                        <i class="fas fa-level-down-alt"></i>
+                    </form>
+                    <div class="full-list"></div>
+                </div>
+            </div>
```

Comment:
Kindly note that everything will be done from the index.js file, so consider removing all these markups here and perform the steps as listed in the project requirement so you can be on same page.